packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -1035,7 +1035,7 @@ export const prepareVaultManagerKit = (
             state.collateralBrand,
           );
           if (!storedCollateralQuote)
-            throw Fail`lockOraclePrices called before a collateral quote was available`;
+            throw Fail`lockOraclePrices called before a collateral quote was available for ${state.collateralBrand}`;
```

Comment:
Is the vault factory getting upgraded in this ? If not, this change should not be part of the dev branch

Reply:
dropped